### DISCUSSION TITLE
read-only user permissions?

### DISCUSSION THREAD

**Discussion by tchwpkgorg** [postalserver/postal#1482] [Feature suggestions]
Users which have "do not grant admin access" permissions are still able to add/remove domains and change other settings.

It should be possible to add more granular permissions, i.e. ones which do not allow any configuration changes.

**Comment by willpower232** [comment]
That would be a good feature yes. Unfortunately there is minimal management of users at the minute so this will have to come after the ability to actually manage users as they currently stand #25

**Reply by qhgh** [reply]
This would be great indeed. Is there any ETA for it?

**Reply by bendem** [reply]
Currently, there are two user types: 
* full admins 
* organisation admins

It would be great to have a third type that would be able to read settings but not modify them (edit or delete). In our case, what we are interested in is:
* can read mail queues
* can send test mails (optional)
* cannot read credential secrets
* cannot edit anything

**Comment by ofekifergan** [comment]
any update?
